Sources/Commands/SwiftPackageTool.swift

```diff
@@ -290,22 +290,41 @@ extension SwiftPackageTool {
     
     struct APIDiff: SwiftCommand {
         static let configuration = CommandConfiguration(
-            commandName: "experimental-api-diff")
+            commandName: "experimental-api-diff",
+            abstract: "Diagnose API-breaking changes to Swift modules in a package",
+            discussion: """
+            The experimental-api-diff command can be used to compare the Swift API of \
+            a package to a baseline revision, diagnosing any breaking changes which have \
+            been introduced. By default, it compares every Swift module from the baseline \
+            revision which is part of a library product. If this behavior is undesirable, \
```

Comment:
Please add a comment on the performance, for example, "This can result in a slow performance if there are a large number of modules. If this behavior is ..."

Reply:
👍 I reworded this to add the extra note. I also ran a couple of benchmarks locally, and it looks like performance probably won't be a major concern for any package with < 100-200 targets. I've been primarily testing with swift-nio, which has about a dozen, and the diff operation takes under 2 seconds using a release w/ asserts frontend